bambooContentScript.js

```diff
@@ -106,6 +120,36 @@ function main() {
                 'duration': timeEntryTotal
             };
 
+            // NOTE: Special case, Saturdays get two entries in spreadsheet.
+            if (isSaturday) {
+                didHaveSatEntry = true;
+
+                if (!previous) {
+                    const BREAK_END = "13:00"
+                    const satBreakTime = `${currentEntry.end} - ${BREAK_END}`;
+                    const satBreakDuration = subtractTimes(BREAK_END, currentEntry.end);
+
+                    htmlString +=
+                        `<tr>
+                            <td>${currentEntry.start}</td>
+                            <td>${currentEntry.end}</td>
+                            <td>${satBreakDuration}</td>
+                            <td>${satBreakTime}</td>
+                        </tr>`.replace(/\s+/g, '');
+
+                    console.log({ satBreakTime, satBreakDuration });
+
+                    previous = currentEntry;
+                } else {
+                    beginning = currentEntry.start;
+                    end = currentEntry.end;
+                    breakTime = "";
+                    breakDuration = "";
+                }
+
+                continue;
```

Comment:
use if else and proper conditions for code flow

Reply:
i think it's slightly cleaner with the `continue`, to avoid an extra indentation for the time calc logic for regular weekdays

Reply:
the indentation is definitely worth it continue is just a another incarnation of goto. The problem is that you use continue/break/etc. you can't use code blocks for navigating the code. Instead you to have read line by line to figure out whats happening.